cypress/support/pages/rooms/pageRooms.js

```diff
@@ -286,6 +292,10 @@ class Rooms {
 
 	seeInfoTextBanner() {
```

Comment:
Since this is a very specialised method to check a defined Info box but not an info box in general, I would suggest to call it "seeInfoBannerForAddingRoomMembers"

Reply:
done